templates/taxi/driver_list.html

```diff
@@ -6,7 +6,11 @@ <h1>Driver list</h1>
     <ul>
       {% for driver in driver_list%}
         <li>
-          <a href="{% url "taxi:driver-detail" pk=driver.id %}">{{ driver.username }}</a>
+          {% if user.pk == driver.pk%}
+            <a href="{% url "taxi:driver-detail" pk=driver.id %}">{{ driver.username }} (Me)</a>
+          {% else %} 
+            <a href="{% url "taxi:driver-detail" pk=driver.id %}">{{ driver.username }}</a>
+          {% endif %}
```

Comment:
Remove the code duplication, it would be better to add the `(Me)` part to the username depending on the driver id rather than duplicating the whole line for that.